libs/coin-evm/src/api/explorer/index.ts

```diff
@@ -26,6 +26,7 @@ export const getExplorerApi = (currency: CryptoCurrency): ExplorerApi => {
   switch (apiType) {
     case "etherscan":
     case "blockscout":
+    case "teloscan":
```

Comment:
Curious about this: what is the point for us of defining three types if they all target the same implem?

Reply:
Indeed unit tests cover the same file seeing https://github.com/LedgerHQ/ledger-live/pull/3827

But they can complement each other, we can resolve conflicts together if this PR happens to be merged before NFT support 👍 , same on the other way around